contracts/instruments/RibbonThetaVault.sol

```diff
@@ -159,15 +162,10 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
      * @notice Closes the vault and makes it withdraw only.
      */
     function sunset(address upgradeTo) external onlyOwner {
-        require(!isSunset, "!isSunset");
-        require(replacementVault == address(0), "!replacementVault");
+        require(address(replacementVault) == address(0), "Already sunset");
         require(upgradeTo != address(0), "!upgradeTo");
 
-        cap = 0;
-        isSunset = true;
-        instantWithdrawalFee = 0;
-        replacementVault = upgradeTo;
-        IVaultUpgrade = IRibbonV2Vault(upgradeTo);
+        replacementVault = IRibbonV2Vault(upgradeTo);
```

Comment:
We're able to set the IRibbonV2Vault instance and replacementVault in one go, instead of setting 2 vars